change/@fluentui-react-tooltip-9e0085eb-2ad8-4d35-9721-29b6f30fc9fe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Remove `inverted` prop that was added back by a merge conflict",
```

Comment:
Can you please extract this change into a separate PR?

- it's not related to changes that this PR proposes
- it creates an entry in a changelog that does not match changes in the PR